src/api/call.ts

```diff
@@ -23,15 +23,19 @@ const buildQueryString: BuildQueryString = R.pipe(
   R.concat('?')
 );
 
+export const buildQueryString = (data: GetParams) =>
+  R.isEmpty(data) ? '' : buildDumbQueryString(data);
+
```

Comment:
If we want to avoid `buildDumbQueryString` we can write :

```ts
type BuildQueryString = (params: GetParams) => string;
const buildQueryString: BuildQueryString = R.ifElse(
  R.isEmpty,
  R.always(''),
  R.pipe(
    R.toPairs,
    R.map(([key, value]: R.KeyValuePair<string, GetParamValue>) =>
      Array.isArray(value)
        ? value
            .map(
              (valueItem: string) =>
                `${encodeURIComponent(key)}[]=${encodeURIComponent(valueItem)}`
            )
            .join('&')
        : `${encodeURIComponent(key)}=${encodeURIComponent(value)}`
    ),
    R.join('&'),
    R.concat('?')
  )
);
```

Reply:
:+1: